deployment/sriov-network-operator/values.yaml

```diff
@@ -25,9 +25,56 @@ operator:
   nameOverride: ""
   fullnameOverride: ""
   resourcePrefix: "openshift.io"
-  enableAdmissionController: false
```

Comment:
before this change, users needed to pre-create tls secrets with predefined names.
just setting this to true would not really work right ?

we breaking (helm) API here, but i think its acceptable.
can you add a check in template to error if this is set to true ?
(pointing out to use admissionControllers.enabled)

see : https://masterminds.github.io/sprig/flow_control.html

Reply:
Hmm, based on the tests I've added above in the description, it looks like for case 3 which is what you are referring to, we didn't break the API. You just need to set one value to true like you used to. It's just that the value has moved in a deeper layer (`operator.admissionControllers.enabled=true` vs `operator.enableAdmissionController`)

Reply:
> It's just that the value has moved in a deeper layer

so it breaks API in helm, upgrading chart with an exiting value override will cause admission controllers to move from enabled to disabled.

Reply:
If that was the original question, then yes, it breaks the API which means we need to bump the major version of the chart (that is not published to my knowledge (?)) on the next release. 

But I thought the original question was if by setting the `operator.admissionControllers.enabled=true` we still get the same behavior with the predefined secrets, which we do.

Reply:
after some offline discussions, it seems that in nvidia network operator we never used this api so no need to handle it for now as we are not aware of any other consumer of helm chart.